scripts/bw2h5.py

```diff
@@ -70,151 +36,113 @@
 _logger.addHandler(_handler)
 
 
-def parse_args():
-    """Parse command line arguments.
-
-    Return:
-        args : parsed argument object.
+def bw2h5(noisybw, cleanbw, layersbw, cleanpeakbw, batch_size, nolabel,
```

Comment:
how about moving this into `atacworks/io` folder since we call it as an API now?

Reply:
I'm vary of moving it into atacworks/io right now. These "modules" still create files and the module returns a path to the output file instead of an object. I feel like it should qualify as a proper module to graduate into atacworks/io. 

I wanted to return output files instead of objects for 2 reasons: 
1) It's a minimal change, that still let's us test the intermediate files through end-to-end testing.
2) Keeping deadline in mind, this is was the fastest way with least potential errors.

Thoughts?

Reply:
After discussing offline, decided to move into an `atacworks` module to enable better testing and prepping for future improvements as an API